app/models/concerns/curation_concerns/solr_document_behavior.rb

```diff
@@ -123,6 +123,10 @@ def read_groups
       fetch(Hydra.config.permissions.read.group, [])
     end
 
+    def source
+      self[Solrizer.solr_name('source')]
```

Comment:
Should this method return `nil` or `[]` when there are no 'source' keys? I wonder if the `fetch()` pattern from above might be a better fit here.